src/test/java/bio/terra/common/fixtures/ConnectedOperations.java

```diff
@@ -124,7 +124,7 @@ public void stubOutSamCalls(IamProviderInterface samService) throws Exception {
 
         when(samService.createSnapshotResource(any(), any(), any())).thenReturn(snapshotPolicies);
         when(samService.isAuthorized(any(), any(), any(), any())).thenReturn(Boolean.TRUE);
-        when(samService.createDatasetResource(any(), any())).thenReturn(datasetPolicies);
+        when(samService.addDatasetResourcePolicies(any(), any())).thenReturn(datasetPolicies);
```

Comment:
@pshapiro4broad  - Do I still need to include the createDatasetResource() call? I'm not really familiar with the stub methods.

Reply:
Sorry I just saw this. You should be OK, `samService` is initialized as a `MockBean` which means that all methods are mocked as no-ops by default. There is code here that uses `doNothing()` but I don't know why it does that. I've only used `doNothing()` before on `spy()` objects.

Reply:
No problem - thanks for following up! With my latest changes, it should also be back to the original code.